tests/integration/setup-rendering-context-test.js

```diff
@@ -119,18 +119,10 @@ module('setupRenderingContext "real world"', function(hooks) {
     test('can click on a sibling element of this.element (within the rootElement)', async function(assert) {
       let rootElement = document.getElementById('ember-testing');
 
-      assert.notEqual(
-        rootElement,
-        this.element,
-        'precond - confirm that the rootElement is different from this.element'
-      );
-
       this.set('rootElement', rootElement);
 
       await render(hbs`{{#-in-element rootElement}}{{click-me-button}}{{/-in-element}}`);
 
-      // this will need to be modified / removed once RFC280 lands
-      assert.equal(this.element.textContent, '', 'no content is contained _within_ this.element');
```

Comment:
why is this removed?

Reply:
Because `this.element` (when the optional flag is enabled) _will_ have content. I can swap things around a bit and guard the assertions, but I think I'd rather have two tests instead.  One for "with a wrapper div" and one "without a wrapper"...

Reply:
what content will be in there and why wasn't it in there before? I'm not sure I understand the assertion 🤔

Reply:
The test (as it exists on master) is actually testing a few different things:

* `this.element` within the test context is **not** the `#ember-testing` div
* when rendering _only_ content that is "wormholed" (via `in-element` in this case), the actual `this.element` is still empty (because the wormholed content is somewhere else)
* that we _can_ interact (e.g. click) with elements that are within the `#ember-testing` div (even though they are above the `this.element`).

As you can tell some of these assertions are no longer needed with the changes from emberjs/rfcs#280.  Specifically:

> `this.element` within the test context is **not** the `#ember-testing` div

without the wrapper DIV, the `this.element` actually _is_ `#ember-testing`

>  when rendering _only_ content that is "wormholed", the actual `this.element` is still empty

due to the above, this is not true any longer. `this.element` will have content (the wormholed content)

These two changes are the assertions that were being removed (when you commented). I have changed things around a bit so that we have two different tests (one with an application template wrapper and one without it).

Reply:
> that we can interact (e.g. click) with elements that are within the #ember-testing div (even though they are above the this.element).

given that we use `document.querySelector('#ember-testing > .ember-view')` how is `click()` able to interact with elements above `this.element`? last time I've tried it didn't work with wormholed content (yet) 🤔

Reply:
This behavior was changed in https://github.com/emberjs/ember-test-helpers/pull/295. `click` and friends are always operating on the root element (`#ember-testing` generally speaking) which is **not** `this.element`.  The specific test that we are commenting on ATM is actually introduce _in_ that PR specifically to address the wormholed content scenario...

Reply:
I see. Turns out we are still on v0.7.10 of `@ember/test-helpers` (though newest `ember-cli-qunit`) which is the reason why it didn't work when I tried it.